.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+language: go
+go_import_path: go.uber.org/goleak
+
+go:
+  # TODO(prashant): Enable once we have fixed Linux issues with old versions.
+  #- 1.6
+  #- 1.7
+  #- 1.8
+  - 1.9
```

Comment:
`1.9.x` ?

Reply:
I thought `gimme` would get us the latest patch release if we specified `1.9`, but that doesn't seem to be the case.

We might have to specifically out the patch version in all our Travis files, cc @abhinav.

Thanks @cristaloleg!